magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RecordPersistence.scala

```diff
@@ -777,7 +820,7 @@ object DefaultRecordPersistence
         // we don't end up with an extraneous record creation event in the database.
         case None =>
           DB.localTx { nested =>
-            createRecordAspect(nested, tenantId, recordId, aspectId, newAspect)
+            createRecordAspect(nested, tenantId, recordId, aspectId, newAspect, config, true)
```

Comment:
Why force skip validation here?

Reply:
We've done the validation earlier (a few lines above) and try to avoid re-do the validation.

Reply:
Ah right - that could use a comment :)

Reply:
Added more comments~